simple/components/restservice/config/cloudify-restservice.service

```diff
@@ -6,8 +6,10 @@ PIDFile=/var/run/gunicorn.pid
 TimeoutStartSec=0
 Restart=on-failure
 EnvironmentFile=-/etc/sysconfig/cloudify-restservice
+User=rest
+Group=rest
 ExecStart=/bin/sh -c '/opt/manager/env/bin/gunicorn \
-    --pid /var/run/gunicorn.pid \
+    --pid /var/run/gunicorn/gunicorn.pid \
```

Comment:
odds are slim.. but we need to make sure this file isn't being read anywhere from within our code.